tests/integration/broken_models/small_office/bad_callback.osw

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I admit that I'm running out of puff, but although I'm may not be catching all of the nuance on these tests, I'm really glad they are here. :)